frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -0,0 +1,150 @@
+  "Conditions": "Conditions",
+  "No Operator Subscription": "No Operator Subscription",
+  "This Operator will not receive updates.": "This Operator will not receive updates.",
+  "All Instances": "All Instances",
+  "{{item}} details": "{{item}} details",
+  "Cluster-wide {{resource}}": "Cluster-wide {{resource}}",
+  "Catalog will be available in all namespaces": "Catalog will be available in all namespaces",
+  "Namespace {{resource}}": "Namespace {{resource}}",
+  "Catalog will only be available in a single namespace": "Catalog will only be available in a single namespace",
+  "Create {{resource}}": "Create {{resource}}",
+  "Create a CatalogSource in order to make operators available in OperatorHub.": "Create a CatalogSource in order to make operators available in OperatorHub.",
+  "{{resource}} name": "{{resource}} name",
+  "Display name": "Display name",
+  "Publisher name": "Publisher name",
+  "Image (URL of container image)": "Image (URL of container image)",
+  "URL of container image hosted on a registry": "URL of container image hosted on a registry",
+  "Create": "Create",
+  "Cancel": "Cancel",
+  "operator": "operator",
+  "operator_plural": "operators",
+  "Installation namespace <1>{{namespace}}</1> contains <4></4> installed with manual approval, and all operators installed in the same namespace will function as manual approval strategy. To allow automatic approval, all operators installed in the namespace must use automatic approval strategy.": "Installation namespace <1>{{namespace}}</1> contains <4></4> installed with manual approval, and all operators installed in the same namespace will function as manual approval strategy. To allow automatic approval, all operators installed in the namespace must use automatic approval strategy.",
+  "Review the manual install plan for operators <1></1>. Once approved, the following resources will be created in order to satisfy the requirements for the components specified in the plan. Click the resource name to view the resource in detail.": "Review the manual install plan for operators <1></1>. Once approved, the following resources will be created in order to satisfy the requirements for the components specified in the plan. Click the resource name to view the resource in detail.",
+  "No InstallPlans found": "No InstallPlans found",
+  "InstallPlans are created automatically by subscriptions or manually using the CLI.": "InstallPlans are created automatically by subscriptions or manually using the CLI.",
+  "Components": "Components",
+  "Subscriptions": "Subscriptions",
+  "InstallPlans": "InstallPlans",
+  "Inspect the requirements for the components specified in this InstallPlan before approving.": "Inspect the requirements for the components specified in this InstallPlan before approving.",
+  "Preview InstallPlan": "Preview InstallPlan",
+  "InstallPlan details": "InstallPlan details",
+  "Review manual InstallPlan": "Review manual InstallPlan",
```

Comment:
The string is `Review Manual Install Plan`
<img width="1001" alt="Screen Shot 2020-12-15 at 6 38 51 PM" src="https://user-images.githubusercontent.com/12692381/102204631-0a973480-3f05-11eb-81bc-d9962544da2c.png">